apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -45,71 +26,41 @@ require('/views/shared/test/unit/mock_utils.js');
 require('/views/shared/js/activity_handler.js');
 
 var mocksHelperForActivityHandler = new MocksHelper([
+  'ActivityClient',
+  'ActivityShim',
   'Attachment',
-  'Contacts',
   'Draft',
   'Drafts',
   'MessageManager',
-  'NotificationHelper',
-  'Notify',
-  'Settings',
-  'SettingsURL',
-  'SilentSms',
-  'SMIL',
-  'Threads',
-  'Utils',
   'Navigation',
-  'ActivityShim',
-  'ActivityClient'
+  'Settings',
+  'Utils'
 ]).init();
 
 suite('ActivityHandler', function() {
   mocksHelperForActivityHandler.attachTestHelpers();
 
-  var realSetMessageHandler;
-  var realWakeLock;
-  var realMozApps;
-  var realMozL10n;
-
-  var isDocumentHidden;
+  var navigatorMocks = new Map([
+    ['mozSetMessageHandler', () => {}]
+  ]);
 
   suiteSetup(function() {
-    realSetMessageHandler = navigator.mozSetMessageHandler;
-    navigator.mozSetMessageHandler = MockNavigatormozSetMessageHandler;
-
-    realWakeLock = navigator.requestWakeLock;
-    navigator.requestWakeLock = MockNavigatorWakeLock.requestWakeLock;
-
-    realMozApps = navigator.mozApps;
-    navigator.mozApps = MockNavigatormozApps;
-
-    realMozL10n = navigator.mozL10n;
-    navigator.mozL10n = MockL10n;
-
-    Object.defineProperty(document, 'hidden', {
-      configurable: true,
-      get: function() {
-        return isDocumentHidden;
+    var mockedNavigatorKeys = navigator[Symbol.for('mocks')] = [];
```

Comment:
Heh, I just declared it for the usage inside loop below :) But agree, I can use ```navigator[Symbol.for('mocks')].push(key)``` directly, since it should not affect anything - it's just test code :)